distributed/worker.py

```diff
@@ -4020,9 +4022,8 @@ async def run(server, comm, function, args=(), kwargs=None, is_coro=None, wait=T
     pass
 else:
 
-    @gen.coroutine
-    def gpu_metric(worker):
-        result = yield offload(nvml.real_time)
+    async def gpu_metric(worker):
+        result = await offload(nvml.real_time)
```

Comment:
@charlesbluca @quasiben 

I think that this change should be a no-op and should not affect you.  However, it's rare to see the use of `gen.coroutine` these days and so maybe this was intentional on the part of whoever wrote this (maybe me?).  I thought I would give you a heads up just in case people start to mention that gpu metrics have started to break.

Reply:
Thanks for the heads up!